src/commands/clone.js

```diff
@@ -41,10 +41,34 @@ function clone (opts) {
   var onExit = require('../lib/exit')
   var parseArgs = require('../parse-args')
   var debug = require('debug')('dat')
-
   var parsed = parseArgs(opts)
+  var path = require('path')
```

Comment:
Can you add the node core modules as the top requires (in this case, right under `require('fs')`?